src/systemTest/java/soot/testing/framework/AbstractTestingFramework.java

```diff
@@ -112,6 +113,7 @@ protected SootMethod prepareTarget(String targetMethodSignature, Collection<Stri
     mockStatics();
     SootMethod sootTestMethod = createTestTarget(targetMethodSignature);
     runSoot();
+    Assert.assertNotNull("Could not find target method. System test setup seems to be incorrect.", sootTestMethod);
```

Comment:
Can you please add something like: "Please try to re-run `mvn test-compile` to make sure that the target code is present for analysis..."